ci/observabilitySREsmoke_tests.sh

```diff
@@ -3,8 +3,12 @@
 set -e
 
 QUALIFIED_VERSION="$(.buildkite/scripts/common/qualified-version.sh)"
-export ELASTICSEARCH_IMAGE_VERSION="${ELASTICSEARCH_IMAGE_VERSION:-$QUALIFIED_VERSION}"
-export FILEBEAT_IMAGE_VERSION="${FILEBEAT_IMAGE_VERSION:-$QUALIFIED_VERSION}"
+# Look up corresponding LOSTASH_STREAM to test against. MAKE SURE TO UDPATE this per branch for example the 9.2 branch in logstash would
+# correspond to the '9.current' key in the logstash-versions.yml file
+LOGSTASH_STREAM=main
```

Comment:
With this initial step no concern and for the long term it would be great if we can dynamically resolve artifact version (eg: if branch 9.1, consider values of 9.prev or 9.current if matches 9.1)

Reply:
I think ultimately this is going to be a "moving target". IE just recently we had to update `9.1 -> 9.previous` `9.2 -> 9.current` so computing from branch name wont be possible without an external input. The other thought I had was to centralize this to look up programatically. Perhaps a new section in `logstash-versions.yml`:

```
logstash-streams:
  "8.19": "8.previous"
  "9.1": "9.previous"
  "9.2": "9.current"
  "main": "9.next"
```